php/Terminus/AnnotatedCommands/ExampleCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Terminus\AnnotatedCommands;
+
+use Robo\ResultData;
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputAwareInterface;
+use Robo\Contract\OutputAwareInterface;
+use Robo\Common\OutputAwareTrait;
+use Robo\Common\InputAwareTrait;
+
+// TODO: add input and output aware interfaces
+class ExampleCommand implements InputAwareInterface, OutputAwareInterface
+{
+    use InputAwareTrait;
+    use OutputAwareTrait;
+
+    /**
+     * @command example-table
+     *
+     * Demonstrate Robo formatters.  Default format is 'table'.
+     *
+     * @field-labels
+     *   first: I
+     *   second: II
+     *   third: III
+     * @usage try:formatters --format=yaml
+     * @usage try:formatters --format=csv
+     * @usage try:formatters --fields=first,third
+     * @usage try:formatters --fields=III,II
+     */
+    public function exampleTable($options = ['format' => 'table', 'fields' => ''])
```

Comment:
Terminus commands parameters take an array with the arguments, and an array with the options.  Robo will provide parameters in the same format, if you wish; you simply need to typehint the first parameter as an `array`, and Robo will pass all arguments in the first parameter. By default, though, Robo passes all of the arguments as individually-named parameters; the last parameter is always the options array.